content/living_abroad.md

```diff
@@ -1,66 +1,49 @@
----
-description: What you can do with your UK pension pot if you move or already live overseas.
```

Comment:
We should have a description. This is used by search engines to explain the content of the page in search results.